tests/test_discord_bot.py

```diff
@@ -0,0 +1,104 @@
+from unittest.mock import AsyncMock
+from unittest.mock import Mock, patch
+from unittest import TestCase
+import pytest
+import json
+
+from gerousiabot import utils
+
+from gerousiabot import discord_bot
+
+logger = utils.setup_logger()
+
+
+bot_token = utils.get_env_variable('DISCORD_API_KEY')
```

Comment:
Since it's being used by all the test cases, let's place this in a setup method that's run before each test case.
This implementation may run into test errors caused by the tests dependencies on each other